### PR TITLE
Issue218

### PR DESCRIPTION
**Pull request by rebeccaSimmonds19**
https://github.com/radanalyticsio/oshinko-s2i/issues/218

### CODE REVIEW

**Comment by rebeccaSimmonds19** [comment]
@crobby this should work on all os it is not specific to macos. Did you try it yourself?

**Comment by crobby** [comment]
Question about the Dockerfile change.  Any changes to the Dockerfile without a change to the corresponding image.yaml will wind-up being overwritten the next time image.yaml changes (and the Dockerfile is regenerated).  Should there be a change to image.java.yaml instead (which will result in the Dockerfile change too)

File: java-build/Dockerfile [hunk 1]
```diff
@@ -26,7 +26,8 @@ ENV JBOSS_IMAGE_NAME="radanalyticsio/radanalytics-java-spark" \
     SPARK_HOME="/opt/spark" \
     APP_ROOT="/opt/app-root" \
     STI_SCRIPTS_PATH="/usr/local/s2i" \
-    APP_LANG="java" 
+    APP_LANG="java"
```
Comment (crobby) [inline java-build/Dockerfile]: Need a \ after this line

**Comment by elmiko** [comment]
just so i understand the change here, if i build this s2i, then make an app from the resulting image i should be able to use the `SPARK_VERSION` environment variable in my pom file?

**Comment by rebeccaSimmonds19** [comment]
yep - fingers crossed!

**Pull request merged**